support/editor-document.js

```diff
@@ -50,6 +59,22 @@ class EditorDocument {
     }
   }
 
+  _removeTemplateComments() {
+    const dom = this.getDom();
```

Comment:
Shouldn't `resetDOM` be used here? In order to also clear `this.topDomNode`?

Reply:
In principle not, because it gets called in the the constructor only.
However, using resetDOM would be the safer and more logical choice, so I will change :+1: